### DISCUSSION TITLE
Auto-screening

### DISCUSSION THREAD

**Discussion by tthui88** [asreview/asreview#621] [Q&A] [answered]
After manual screening on the ASReview Lab interface, how can we start the auto-screening process using machine learning? I don't find it on any documentation, please advise.

**Comment by govertv** [comment, marked as answer]
Nice to hear that you are using ASReview! We believe that it is important that you see each document that gets included into your systematic review during the screening phase, so there is no fully automated screening. 

ASReview can speed up your screening process by controlling the order in which documents are presented to you. A machine learning model is trained in the background, calculating relevance scores for all the documents that you did not see yet. With the default settings, the document that is most likely to be relevant is shown next. This saves you time, because you will not have to screen every record in your dataset.

You might find this image from the documentation helpful: 
https://asreview.readthedocs.io/en/latest/_images/FlowChartC.png

**Reply by tthui88** [reply]
Thanks for the clarification, I was thinking whether an autonomous screening process might be able to label those unscreened records without user intervention.
